### ISSUE TITLE
Create helpful editor hints/warnings for accessibility

### ISSUE THREAD

**Issue by willnationsdev** [dialogic-godot/dialogic#5]
Just taking a look at Mark Brown's [Designing for Disability](https://www.youtube.com/watch?v=4NGe4dzlukc&list=PLc38fcMFcV_vvWOhMDriBlVocTZ8mKQzR) series where he discusses subtitles and the like. I figure you can do a couple of things, like...

- have larger paragraphs of text clearly indicate where breakpoints should be added to split it into a separate section in order to ensure the font size fits a limited amount of text on screen for readability.
- design tools that conduct color tests on the font/background/scene background to ensure that the font is clearly legible at all times.

This issue is just to throw some ideas out there. Could be used to catalog other more specific issues of implementing a particular solution for one of the problems mentioned in Mark's YouTube series.

**Comment by coppolaemilio** [comment]
I'm going to close this one because it is too broad. It would be nice to make some improvements with accessibility in mind, but there should be individual issues instead of a general one. Thanks for opening it! I always felt bad about closing it but it is really not actionable at the moment.
